concourse/pipelines/create-bosh-concourse.yml

```diff
@@ -1,4 +1,47 @@
 ---
+meta:
+  containers:
+    awscli: &awscli-image-resource
+      type: docker-image
+      source:
+        repository: governmentpaas/awscli
+        tag: b2495d6ed07f680125d19aa7d1701da7efabb289
+    bosh-cli-v2: &gov-paas-bosh-cli-v2-image-resource
+      type: docker-image
+      source:
+        repository: governmentpaas/bosh-cli-v2
+        tag: 93e168b4244873902cd552ae08fc4a67a7b2e6f5
+    certstrap: &certstrap-image-resource
+      type: docker-image
+      source:
+        repository: governmentpaas/certstrap
+        tag: 895cf6752c8ec64af05a3a735186b90acd3db65a
```

Comment:
We had two different tags previously and 465642da06051a55630d39c899697b678f66a7f7 is the newer one, so I think we should use that.

Reply:
I picked the one that was used in more places. If you look at [the diff](https://github.com/alphagov/paas-docker-cloudfoundry-tools/compare/895cf67...465642d), you can see that there are actually no changes to the certstrap container between those versions.